src/cpu.h

```diff
@@ -95,6 +95,8 @@ NCNN_EXPORT int cpu_support_x86_avx512_fp16();
 
 // msa = mips mas
 NCNN_EXPORT int cpu_support_mips_msa();
+// msa = loongarch64 lsx
+NCNN_EXPORT int cpu_support_loongarch64_lsx();
```

Comment:
loongarch is a new base arch, place them in new group instead of messing into mips ones
add cpu_support_loongarch_lasx() for lasx

Reply:
fix